modules/metrics/metrics.py

```diff
@@ -0,0 +1,266 @@
+from operator import itemgetter
+import pycountry
+
+
+class Metric:
+    def __init__(
+            self, name, series, buckets, status):
+        self.name = name
+        self.series = series
+        self.buckets = buckets
+        self.status = status
+
+    def to_dictionnary(self):
```

Comment:
You could refer to this through builtin functions, or use them directly:
```
def __dict__(self):
    return self.to_dictionary()
```

So that `Dict(metric_object)` works

Reply:
done